metadata.json

```diff
@@ -73,6 +73,12 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Gentoo",
+      "operatingsystemrelease": [
+        "1.0"
```

Comment:
Ok, so can you do a bit of research and make sure you're happy with what goes in there, then let me know and I'll merge?